src/components/events-list/event-summary.js

```diff
@@ -50,7 +50,7 @@ const EventSummary = ({ event }) => (
 );
 
 EventSummary.propTypes = {
-  event: PropTypes.shape(eventModel).isRequired,
+  event: PropTypes.shape(EventModelValidationMap).isRequired,
 };
 
 export default EventSummary;
```

Comment:
The default export is not used anywhere and can be removed

Reply:
Thanks for the observation. I will have this fixed